### PR TITLE
add email column

### PR DESCRIPTION
**Pull request by sinisaos**
Related to #593

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -5,3 +5,4 @@ targ>=0.3.7
 inflection>=0.5.1
 typing-extensions>=3.10.0.0
 pydantic>=1.6
+email-validator>=1.2.1
```
Comment (dantownsend) [inline requirements/requirements.txt:8]: Can we do anything like `pydantic[email]` instead?

Comment (sinisaos) [reply requirements/requirements.txt:8]: Yes. I will change that.

Comment (dantownsend) [reply requirements/requirements.txt]: Cool, thanks

**Comment by dantownsend** [comment]
@sinisaos I made some minor tweaks. When the tests pass I'll release it.

**Pull request merged**